android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -57,44 +52,50 @@
 // clang-format off
 @Kroll.proxy(propertyAccessors = {
 	// background properties
-	"backgroundImage",
-	"backgroundRepeat",
-	"backgroundSelectedImage",
-	"backgroundFocusedImage",
-	"backgroundDisabledImage",
-	"backgroundColor",
-	"backgroundSelectedColor",
-	"backgroundFocusedColor",
-	"backgroundDisabledColor",
-	"backgroundPadding",
-	"backgroundGradient",
+	TiC.PROPERTY_BACKGROUND_IMAGE,
+	TiC.PROPERTY_BACKGROUND_REPEAT,
+	TiC.PROPERTY_BACKGROUND_SELECTED_IMAGE,
+	TiC.PROPERTY_BACKGROUND_FOCUSED_IMAGE,
+	TiC.PROPERTY_BACKGROUND_DISABLED_IMAGE,
+	TiC.PROPERTY_BACKGROUND_COLOR,
+	TiC.PROPERTY_BACKGROUND_SELECTED_COLOR,
+	TiC.PROPERTY_BACKGROUND_FOCUSED_COLOR,
+	TiC.PROPERTY_BACKGROUND_DISABLED_COLOR,
+	TiC.PROPERTY_BACKGROUND_PADDING,
+	TiC.PROPERTY_BACKGROUND_GRADIENT,
 	// border properties
-	"borderColor", "borderRadius", "borderWidth",
+	TiC.PROPERTY_BORDER_COLOR,
+	TiC.PROPERTY_BORDER_RADIUS,
+	TiC.PROPERTY_BORDER_WIDTH,
 	// layout / dimension (size/width/height have custom accessors)
-	"left", "top", "right", "bottom", "layout", "zIndex",
+	TiC.PROPERTY_LEFT,
+	TiC.PROPERTY_TOP,
+	TiC.PROPERTY_RIGHT,
+	TiC.PROPERTY_BOTTOM,
+	TiC.PROPERTY_LAYOUT,
+	TiC.PROPERTY_ZINDEX,
 	// accessibility
 	TiC.PROPERTY_ACCESSIBILITY_HINT,
 	TiC.PROPERTY_ACCESSIBILITY_LABEL,
 	TiC.PROPERTY_ACCESSIBILITY_VALUE,
 	TiC.PROPERTY_ACCESSIBILITY_HIDDEN,
 	// others
-	"focusable",
-	"touchEnabled",
-	"visible",
-	"enabled",
-	"opacity",
-	"softKeyboardOnFocus",
-	"transform",
-	"elevation",
-	"touchTestId",
```

Comment:
This change is removing "touchTestId", but that's probably fine since it's not referenced anywhere else in the code. Our iOS code doesn't have any references to it either.